src/lib/components/dialog/QDialog.svelte

```diff
@@ -19,41 +18,51 @@
   const emit = createEventDispatcher();
   let dialogElement: HTMLDialogElement;
 
-  $: canHideOnClickOutside = value === true && persistent !== true;
+  let opened = false;
+
+  onMount(() => {
+    opened = dialogElement.open;
+    opened ? (dialogElement.style.display = "block") : (dialogElement.style.display = "none");
```

Comment:
Use the ternary operator instead (`dialogElement.style.display = opened ? 'block' : 'none'`).

Reply:
Yes, my bad, didn't see it lol